Source/WebCore/rendering/RenderTextControl.cpp

```diff
@@ -87,7 +87,7 @@ void RenderTextControl::styleDidChange(StyleDifference diff, const RenderStyle*
 int RenderTextControl::scrollbarThickness() const
 {
     // FIXME: We should get the size of the scrollbar from the RenderTheme instead.
-    return ScrollbarTheme::theme().scrollbarThickness(this->style().scrollbarWidth());
+    return ScrollbarTheme::theme().usesOverlayScrollbars() ? 0 : ScrollbarTheme::theme().scrollbarThickness(this->style().scrollbarWidth());
```

Comment:
This is duplicating logic that `verticalScrollbarIntrusion()` already has. Can we share code? Or, maybe `ScrollbarTheme::theme().scrollbarThickness()` should take an argument to specify how the result should behave for overly scrollbars.

Reply:
That should be fixed now. I pass an argument into scrollbarThickness()